code/ui-utils/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
```

Comment:
Please include ` id("aep-library")` as a plugin here. It already has most of the settings declared in this file. You will only need to include additional dependencies not provided by the library
Example: https://github.com/adobe/aepsdk-ui-android/blob/dev-v3.0.0/code/notificationbuilder/build.gradle.kts